packages/flutter/lib/src/painting/strut_style.dart

```diff
@@ -33,12 +33,17 @@ import 'text_style.dart';
 ///
 /// The sum of these four values is the total height of the line.
 ///
-/// The `ascent + descent` is equivalent to the [fontSize]. Ascent is the font's
-/// spacing above the baseline without leading and descent is the spacing below the
-/// baseline without leading. Leading is split evenly between the top and bottom.
-/// The values for `ascent` and `descent` are provided by the font named by
-/// [fontFamily]. If no [fontFamily] or [fontFamilyFallback] is provided, then the
-/// platform's default family will be used.
+/// Ascent is the font's spacing above the baseline without leading and descent is
+/// the spacing below the baseline without leading. Leading is split evenly between
+/// the top and bottom. The values for `ascent` and `descent` are provided by the
+/// font named by [fontFamily]. If no [fontFamily] or [fontFamilyFallback] is
+/// provided, then the platform's default family will be used.
+///
+/// When `height` is omitted or null, then the font's ascent and descent will be
```

Comment:
If height is a property (which I think it is), this should be [height]. Same for fontSize.